tests/cases/fourslash/findAllRefsOfConstructor.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: f.ts
+
+////class A {
+////    [|constructor|](s: string) {}
+////}
+////class B extends A { }
+////class C extends B {
+////    constructor() {
+////        [|super|]("");
+////    }
+////}
+////class D extends B { }
+////class E implements A { }
+////const a = new [|A|]("a");
+////const b = new [|B|]("b");
+////const c = new C();
+////const d = new [|D|]("d");
+////const e = new E();
+

```

Comment:
Need another test where class C and D extend for B which has constructor declaration